docs/how-to/platform-console.md

```diff
@@ -58,22 +57,49 @@ To create a private plugin:
 2. Click on "Create Private Repository".
 3. Connect to your plugin repository and add your existing private plugins to the project.
 
-## Platform Changes Sidebar
+### Technical Debt Management
+
+The Technical Debt tab helps you track and manage accumulated technical debt across all your services. It provides a comprehensive grid view of services that may be using outdated templates or plugins.
+
+For each service with technical debt, you can see:
+- Alert type (template or plugin updates)
+- Current version in use
+- Latest available version
+- Status (Active, Resolved, or Ignored)
+
+When you click "View Details" for any alert, you'll be taken to that specific service's technical debt tab where you can:
+- View detailed diffs comparing the old and new versions
+- Upgrade to the latest version using the "Upgrade" button
+- Choose to ignore specific updates
+- Resolve technical debt by applying the changes and generating a new service version
```

Comment:
What's the diff between this bullet and the 2nd one (i.e. "Upgrade...")?
Let's unite them